src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -423,6 +423,16 @@ MTR_EXTERN NSString * const MTRDataVersionKey MTR_NEWLY_AVAILABLE;
  */
 - (void)deviceCachePrimed:(MTRDevice *)device MTR_NEWLY_AVAILABLE;
 
+/**
+ * Notifies delegate when the device configuration changes. Device configuration changes include updates in parts list, device type list,
+ * server list, accepted commands list, attribute list, feature map or cluster revision attributes in the descriptor cluster.
```

Comment:
This description is pretty confusing.  I am sure that we are watching for changes to accepted commands list in non-descriptor clusters, yes?

More to the point: should we actually over-specify in the API what "device configuration" means?

Reply:
fixed.